docs/settings/logging-settings.asciidoc

```diff
@@ -36,7 +44,7 @@ logging:
 ==== Log in json format
 
 Log the default log format to json layout instead of pattern (the default).
-With `json` layout log messages will be formatted as JSON strings in https://www.elastic.co/guide/en/ecs/current/ecs-reference.html[ECS format] that includes a timestamp, log level, logger, message text and any other metadata that may be associated with the log message itself
+With `json` layout log messages will be formatted as JSON strings in https://www.elastic.co/guide/en/ecs/1.9/ecs-reference.html[ECS format] that includes a timestamp, log level, logger, message text and any other metadata that may be associated with the log message itself
```

Comment:
Unrelated to the topic at hand, but figured I would update this while I'm touching the file.